### ISSUE TITLE
`minItems` and `maxItems` property is not used

### ISSUE THREAD

**Issue by bruno-f-cruz** [bonsai-rx/sgen#42]
(Adding this issue as a placeholder, will add more details later).

The property seems to be included in the njson API, but for some reason is not trickling down to the sgen generated classes.

https://github.com/RicoSuter/NJsonSchema/blob/1569e52401628d009af8044d1886d5a26770d05c/src/NJsonSchema/JsonSchema.cs#L491

**Comment by bruno-f-cruz** [comment]
I was imagining pigging back on top of these flags: https://www.newtonsoft.com/json/help/html/P_Newtonsoft_Json_Schema_JsonSchema_MinimumItems.htm

Not sure if the yml backend supports something similar.

**Comment by bruno-f-cruz** [comment]
Well the way I see it, I am applying the same standard as we do for "required" properties. These constraints are only applied during (de)serialization, no?

**Comment by glopesdev** [comment]
Yes, but what I mean to say is I am not sure how specifically the `minItems` and `maxItems` constraint is annotated in the object class code.

For example, the `required` attribute needs the following annotation: https://www.newtonsoft.com/json/help/html/JsonPropertyRequired.htm

The links you sent previously are for the JSON schema object model classes but they don't seem to be annotations that can be applied to the object. I would need to know which annotations need to be applied in the case of `minItems` and `maxItems` for both Newtonsoft.Json and YamlDotNet.
